docs/collections.md

```diff
@@ -1,107 +1,90 @@
+- Pagination: `/posts/1`, `posts/2`, etc.
+- Grouping Content by Tag: `/tags/red`, `tags/blue`, etc.
+- Grouping Content by Author: `/posts/fred`, `/posts/matthew`, etc.
+- Working with Remote Data
 
-## 🧑‍🎨 How to Use
+Use a Collection when you need to reuse a single template to generate multiple pages from a single dataset.
 
-By default, any Astro component can fetch data from any API or local `*.md` files. But what if you had a blog you wanted to paginate? What if you wanted to generate dynamic URLs based on metadata (e.g. `/tag/:tag/`)? Or do both together? Astro Collections are a way to do all of that. It’s perfect for generating blog-like content, or scaffolding out dynamic URLs from your data.
 
-Let’s pretend we have some blog posts written already. This is our starting project structure:
+## Collections API
 
-```
-└── src/
-    └── pages/
-        └── post/
-            └── (blog content)
-```
+To create a new Astro Collection, you must do three things:
 
-The first step in adding some dynamic collections is deciding on a URL schema. For our example website, we’re aiming for the following URLs:
-
-- `/post/:post`: A single blog post page
-- `/posts/:page`: A list page of all blog posts, paginated, and sorted most recent first
-- `/tag/:tag`: All blog posts, filtered by a specific tag
+```js
+// 1. Create a new file in the `src/pages` directory that starts 
+// with the `$` symbol. This is required to enable the Collections API.
+// 
+// Example: `src/pages/$posts.astro` would be the correct file name 
+// for generating a colleciton pages using the `/posts/...` URL scheme.
+
+// 2. Define the `collection` prop: `collection.data` is how you'll access 
+// the data for every page in the collection. Astro populates this prop 
+// for you automatically. It MUST be named `collection`.
+export let collection: any;
 
-Because `/post/:post` references the static files we have already, that doesn’t need to be a collection. But we will need collections for `/posts/:page` and `/tag/:tag` because those will be dynamically generated. For both collections we’ll create a `/src/pages/$[collection].astro` file. This is our new structure:
+// 3. Define the `createCollection` call: this tells Astro how to load and
+// structure your collection data. Check out the examples below.
+export async function createCollection() { /* ... */ }
+``` 
 
-```diff
-  └── src/
-      └── pages/
-          ├── post/
-          │   └── (blog content)
-+         ├── $posts.astro     -> /posts/1, /posts/2, …
-+         └── $tag.astro       -> /tag/:tag/1, /tag/:tag/2, …
-```
 
-💁‍ **Tip**: Any `.astro` filename beginning with a `$` is how it’s marked as a collection.
+- `collection` API Refernece: [collection][collection-api].
+- `createCollection()` API Reference: [createCollection][collection-api]
 
-In each `$[collection].astro` file, we’ll need 2 things:
+## Example: Simple Pagination
```

Comment:
If the user wants to have something a page per collection item, the only thing that would need to be changed from this simple pagination example would be the `pageSize`?

Reply:
yes! Great call, that honestly should be an example of its own

Reply:
added!